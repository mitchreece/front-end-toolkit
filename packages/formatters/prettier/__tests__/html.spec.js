const { resolve } = require('path');
const { getFileContent, formatFile } = require('./utils/file');

const filePath = resolve(__dirname, './samples/sample.html');

describe('HTML formatting', () => {
    test('matches snapshot', () => {
        const fileContents = getFileContent(filePath);
        const formattedFile = formatFile(fileContents, { parser: 'html' });

        expect(formattedFile).toMatchSnapshot();
    });
});
