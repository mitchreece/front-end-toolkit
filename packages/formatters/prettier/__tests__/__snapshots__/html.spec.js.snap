// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HTML formatting matches snapshot 1`] = `
"<!DOCTYPE html>
<html lang=\\"en\\">
  <head>
    <meta charset=\\"UTF-8\\" />
    <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0\\" />
    <meta http-equiv=\\"X-UA-Compatible\\" content=\\"ie=edge\\" />
    <title>App Title</title>
  </head>
  <body>
    <div id=\\"root\\"></div>
  </body>
</html>
"
`;
